aries-site/src/components/content/BodyText.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Paragraph } from 'grommet';
+
+// Body text size should be based on if its parent heading is an
+// h2, h3, etc.
+const TEXT_SIZE = {
+  1: 'large',
+  2: 'medium', // default font size
+  3: 'small',
+};
+
+export const BodyText = ({ children, level, size, ...rest }) => {
```

Comment:
I don't think you need to use `children` if you use `...rest`.

Reply:
The children would be what is inside the paragraph, but the `...rest` would be if we wanted to allow customization of other paragraph props. If I remove `children` I get an error.

Reply:
I think you were able to figut=re it out already, marking it as resolved.